### PR TITLE
118 march23 update

### PR DESCRIPTION
**Pull request by wrex**
Creating a pull request because it's at least functional and people are griping about the new changes.

Let's push this ASAP. We can fix remaining nits as we have time, but let's at least get it basically functional.

I've been using it for my reviews without issue. I've also created a new account so I could have some lessons to examine.

I've also tried to only add rules and not delete anything (just in case they decide to revert back to the old stuff, even temporarily).

Looks good enough to go live now, imo, since review/lesson pages are totally broken without these changes.

### CODE REVIEW

**Comment by wrex** [comment]
Ping. 

Any chance you can take a look at this?

File: WKElementaryDark.css [hunk 1]
```diff
@@ -210,6 +210,50 @@
...
+    --color-quiz-incorrect-text-shadow: none;
+    --color-quiz-correct-background: var(--ED-success);
+    --color-quiz-correct-text-color: var(--ED-text);
+    --color-quiz-correct-text-shadow: none;
+    --color-quiz-srs-correct-background: var(--ED-success);
+    --color-quiz-srs-correct-text-color: var(--color-white);
+    --color-quiz-srs-correct-text-shadow: none;
+    --color-quiz-srs-incorrect-background: var(--ED-incorrect);
+    --color-quiz-srs-incorrect-text-color: var(--ED-text);
+    --color-quiz-srs-incorrect-text-shadow: none;
```
Comment (Sepitus-exe) [inline WKElementaryDark.css:256]: This seems dangerous. The amount of testing required to make sure this doesn't result in any unwanted behaviour is making my head spin.

File: WKElementaryDark.css [hunk 2]
```diff
@@ -1479,6 +1602,18 @@
...
+   */
+  .subject-section__subsection
+    .subject-section__meanings:first-child
+    .subject-section__meanings-items,
+  .reading-with-audio__reading {
+    color: var(--ED-text-hl);
+    font-size: x-large;
+    font-weight: bold;
+  }
+
```
Comment (Sepitus-exe) [inline WKElementaryDark.css:1616]: I will probably opt to use filters here down the line. Its a great addition, but something like this should probably be user tweakable.

File: user-overrides/All-variables.css [hunk 3]
```diff
@@ -68,12 +68,12 @@
...
      */
-    --USER-logo-filter: invert(1)saturate(0)brightness(1.6);
+    --USER-logo-filter: invert(1) saturate(0) brightness(1.6);
     --USER-extra-study-filter: invert(0.81);
     --USER-locked-item-filter: brightness(70%);
     --USER-kotoba-odd-row-filter: brightness(1.1);
     --USER-footer-filter: invert(1);
-    --USER-loading-filter: grayscale(90%)invert(1)hue-rotate(180deg);
+    --USER-loading-filter: grayscale(90%) invert(1) hue-rotate(180deg);
     --USER-overlay-box-highlights-filter: hue-rotate(160deg);
```
Comment (Sepitus-exe) [inline user-overrides/All-variables.css:77]: Iirc the spaces between filters cause trouble in some instances. I would like to encourage users to keep the filters spaceless if possible as a precaution. Even at the cost of making the whole string less legible.

Also, note to self: The filters in the override file should be commented out by default. At the time of implementation, I most likely forgot to do so.

**Review by Sepitus-exe — APPROVED** [review]
Super quick review, Im pushing it out as I writte this. Im bound to do some changes once I get some free time on my hands. The legacy code will also needs to be weeded out, but that's an eventual thing rather than an immediate one.

**Pull request merged**